pkg/interface/link/src/js/lib/util.js

```diff
@@ -117,6 +128,8 @@ export function deSig(ship) {
   return ship.replace('~', '');
 }
 
+//TODO look at uxToHex wonky functionality
+//TODO what does "wonky functionality" refer to?
```

Comment:
@loganallenc has more here — essentially I *think* this utility had a weird thing going on where it wasn't returning the hex value set, but usually a digit off? I think it might have had something to do with how its returned value operated with the view? I think Contacts uses this utility, but it might have a fix; I know he removed the `#` from the Sigil Color field entirely at least partly because of this.

Reply:
Yes, please update to use my modified uxToHex util in the contacts JS